uaclient/contract.py

```diff
@@ -15,7 +15,6 @@
     pass
 
 API_ERROR_INVALID_TOKEN = "invalid token"
-API_ERROR_MISSING_INSTANCE_INFORMATION = "missing instance information"
```

Comment:
Does this error now return a 4xx code?

Reply:
Yes this missing instance information error is specifically what was fixed with two other failure conditions to return a 404:
storeutil.ErrAmbiguousProductMapping,
storeutil.ErrMissingProductMapping,
storeutil.ErrMissingInstanceInformation:

More context:
https://github.com/CanonicalLtd/ua-contracts/pull/517/files#r342118256